src/components/Compendia/DownloadBlockForm.js

```diff
@@ -1,34 +1,25 @@
-import { useState, memo } from 'react'
+import { useRef, useState, memo } from 'react'
 import { Box, Heading, Text } from 'grommet'
 import styled, { css } from 'styled-components'
-import gtag from 'analytics/gtag'
-import { links } from 'config'
-import { useCompendia } from 'hooks/useCompendia'
+import { useCompendiaContext } from 'hooks/useCompendiaContext'
 import { useResponsive } from 'hooks/useResponsive'
+import { useToken } from 'hooks/useToken'
 import formatBytes from 'helpers/formatBytes'
 import formatString from 'helpers/formatString'
+import fuzzyFilterOnKey from 'helpers/fuzzyFilterOnKey'
 import getReadable from 'helpers/getReadable'
+import { links } from 'config'
 import { Anchor } from 'components/shared/Anchor'
+import { BoxBlock } from 'components/shared/BoxBlock'
 import { Button } from 'components/shared/Button'
 import { CheckBox } from 'components/shared/CheckBox'
 import { Column } from 'components/shared/Column'
-import { List } from 'components/shared/List'
 import { Icon } from 'components/shared/Icon'
 import { InlineMessage } from 'components/shared/InlineMessage'
 import { Row } from 'components/shared/Row'
 import { SearchBox } from 'components/shared/SearchBox'
 
-const DropDown = styled(Box)`
-  > div:nth-child(2) {
-    display: none;
-  }
-  &:focus-within > div:nth-child(2) {
-    display: block;
-    box-shadow: 0px 3px 4px rgba(0, 0, 0, 0.3);
-  }
-`
```

Comment:
I've replaced this `DropDown` styled-componet  with `useRef` hook to handle the dropdown option toggling (in e248f1f).